katdal/h5datav3.py

```diff
@@ -482,17 +482,17 @@ def register_telstate_sensor(name, obj):
         elif spw_params['band'] == 'UHF' and bandwidth == 856e6:
             spw_params['centre_freq'] = 428e6
             spw_params['sideband'] = -1
-        # If the file has output parameters, use those instead
+        # If the file has SDP output stream parameters, use those instead
         num_chans = self._get_telstate_stream_attr('n_chans', num_chans)
         bandwidth = self._get_telstate_stream_attr('bandwidth', bandwidth)
         stream_centre_freq = self._get_telstate_stream_attr('center_freq')
         if stream_centre_freq is not None:
             spw_params['centre_freq'] = stream_centre_freq
-        # Get channel width from original CBF parameters
+        # Get channel width from original CBF / SDP parameters
         spw_params['channel_width'] = bandwidth / num_chans
         # Continue with different channel count, but invalidate centre freq (keep channel width though)
         if num_chans != self._vis.shape[1]:
-            logger.warning('Number of channels received from ingest (%d) differs '
+            logger.warning('Number of channels received from CBF / ingest (%d) differs '
```

Comment:
Why this change? When does a v3 file ever contain data received from CBF?

Reply:
I was thinking of the source of the information - ingest just passed CBF spead attrs through in the past. Now it can also modify them.

Reply:
Improved.